### ISSUE TITLE
Excessive vertical padding in group title rows

### ISSUE THREAD

**Issue by ilikepeaches** [transmission/transmission#7290]
### What is the issue?

In the first macOS beta of 4.1.0: when "Use Groups" is on, the second and subsequent group title rows are much taller than they should be, including both when the group is open and when it's closed.  The first group appears never to be affected; when the groups are re-ordered, whichever becomes the first group gets the correct title row height.

In these screens, the first title bar row is the correct height:

![screen1](https://github.com/user-attachments/assets/8626da9e-3bda-47ba-b1d2-e1b3087ad28a)

![screen2](https://github.com/user-attachments/assets/0010cd5d-2c3b-45b9-93be-c19060c8e48e)

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.1.0-beta.1 (c3a8106077)

**Comment by hughlilly** [comment]
I agree this padding is excessive. There also seems to have been more horizontal padding added around icons in both the full and compact views, which I also find unnecessary.

**Comment by livings124** [comment]
> There also seems to have been more horizontal padding added around icons in both the full and compact views, which I also find unnecessary.

I've noticed this as well. There's a bunch of subtle visual changes that snuck in when this was rewritten to be view based.

**Comment by livings124** [comment]
I've merged a hopeful fix for this specific issue. The other spacing changes remain and should probably be tracked elsewhere.

**Comment by livings124** [comment]
I made some layout tweaks in https://github.com/transmission/transmission/pull/7327

**Comment by bradtchapman** [comment]
@livings124 @1100101 @ilikepeaches : I can confirm that the spacing issue is resolved in 4.1.0 b2, and this issue can be closed.
